packages/dx-grid-core/src/plugins/table-group-row/computeds.ts

```diff
@@ -114,12 +114,10 @@ export const tableGroupCellColSpanGetter: GroupCellColSpanGetter = (
 
   if (tableRow.type === TABLE_GROUP_TYPE) {
     const colName = tableColumn.column?.name;
-    const dataColumnGroupedBy =
-      tableRow.row.groupedBy === colName && tableColumn.type !== TABLE_GROUP_TYPE;
     const chains = groupSummaryChains(
       tableRow, tableColumns, groupSummaryItems, firstVisibleColumnIndex,
     );
-    const chain = chains.find(ch => !dataColumnGroupedBy && ch[0] === colName);
+    const chain = tableRow.row.groupedBy ? chains.find(ch => ch[0] === colName) : undefined;
```

Comment:
I have an question. 
when "tableRow.type === TABLE_GROUP_TYPE", if this mean "tableRow.row.groupedBy" always have an value?

I think we should only overwrite the colspan of `group type column`

Reply:
I think if there are multiple column type which have same column name,  we should get the colspan according to it's type.
Maybe we should use key of column in chains but not column name, to prevent there are mutiple chains start with same column name.

If a column is data type,  the colspan of it should not be the default value 1,  that will make the number of `td` less than we wanted,  because in some case render `null` instead `templatePlaceholder` in tableGroupRow plugin.